Sources/RInAppMessaging/Utilities/AnalyticsTracker.swift

```diff
@@ -10,7 +10,17 @@ internal struct AnalyticsTracker {
     static func sendEventName(_ name: Constants.RAnalytics, dataObject: [String: Any] = [:]) {
         var eventData = dataObject
         eventData[Constants.RAnalytics.Keys.deviceID] = UIDevice.deviceID
-        AnalyticsBroadcaster.sendEventName(name.rawValue,
-                                           dataObject: eventData)
+        
+        if let customAccNumber = BundleInfo.rmcRATAccountId, RInAppMessaging.isRMCEnvironment {
+            let rmcEventName = name.rawValue == Constants.RAnalytics.impressionsEventName.rawValue ? Constants.RAnalytics.rmcImpressionsEventName : Constants.RAnalytics.rmcPushPrimerEventName
+
+            AnalyticsBroadcaster.sendEventName(rmcEventName.rawValue,
```

Comment:
it can be moved outside the condition & write condition to send only RMC events else condition is not required